Maddori.Apple/Maddori.Apple/Global/UIComponent/LabelButtonView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Label+ButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2022/10/22.
+//
+
+import UIKit
+
+import SnapKit
+
+class LabelButtonView: UIView {
+    
+    var buttonAction: (() -> ())?
+    
+    var subText: String = "" {
+        didSet {
+            subLabel.text = subText
+        }
+    }
+    
+    var subButtonText: String = "" {
+        didSet {
+            subButton.setTitle(subButtonText, for: .normal)
+        }
+    }
+    
+    // MARK: - property
+    
+    private lazy var subButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle(subButtonText, for: .normal)
+        button.titleLabel?.font = .body2
+        let action = UIAction { [weak self] _ in
+            self?.buttonAction?()
+        }
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    private lazy var subLabel: UILabel = {
+        let label = UILabel()
+        label.text = subText
+        label.font = .body2
+        label.textColor = .gray400
+        return label
+    }()
+    
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) { nil }
+    
+    func render() {
```

Comment:
p4;

```suggestion
    private func render() {
```

이게 보다보면 조금씩 보여서 계속 낱개로 남기게되네요.. 죄송합니다...
외부에서 접근할 경우가 없을 것 같은데 `private ` 부탁드려도 될까요 ???

Reply:
[710686d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/32/commits/710686d089e7fc912f482698d404746fcc386b29)

사실 이게 테스트 한다고 만들던 파일이라 아무것도 안지켜서 그런건데요뭐.. ㅠㅠ 제가 잘 했어야죠 !! 남겨줘서 곰아워요~